scripts/check_command_type_numbering.py

```
@@ -0,0 +1,120 @@
+        tok = tok.strip()
+        if not tok:
+            continue
+        name, val = tok.split("=")
+        exp[name.strip()] = int(val)
+    return exp
+
+
+def actual():
+    with open(HEADER) as f:
+        lines = f.read().splitlines()
+    i0 = next(i for i, l in enumerate(lines)
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)